clap_complete/tests/testsuite/dynamic.rs

```diff
@@ -288,6 +289,112 @@ goodbye-world
     );
 }
 
+#[test]
+fn suggest_argument_value() {
```

Comment:
Can you add a test for multiple shorts, like `"-ccccF [TAB]"`

Reply:
Thanks for you review! Now I have added the test.